action.yml

```diff
@@ -81,9 +84,15 @@ runs:
         fi
 
         output=$($oxygen_command --buildCommand="${build_command_filtered}")
-        echo "url=$output" >> $GITHUB_OUTPUT
         exit_code=$?
 
+        if jq -e . >/dev/null 2>&1 <<< $output; then
+          echo $output | jq '. | .url' | awk '{print "url="$1}' >> "$GITHUB_OUTPUT"
+          echo $output | jq '. | .testingAutomationToken' | awk '{print "testing_automation_token="$1}' >> "$GITHUB_OUTPUT"
```

Comment:
Is the end goal to have that token as a query param? Why not just have it as part of url in the output when present.

For example: 

Token present => `url=https://blah.myshopify.dev?<token_key>=<token_value>

Token missing => `url=https://blah.myshopify.dev` (same as today)